src/option.ts

```diff
@@ -480,7 +480,7 @@ export namespace OptionT {
    * @param {T} val
    * @returns {OptT<any>}
    */
-  export function some<T>(val: T) {
```

Comment:
talked about this in keybase, but this shouldnt change, in fact I think it should be `OptT<T>` and should throw an error if not a some.